ext-src/utils/exec.ts

```diff
@@ -14,8 +14,11 @@
  * limitations under the License.
  */
 import * as cp from 'child_process';
+import * as process from 'process';
 
 export function exec(command: string, options: cp.ExecOptions): Promise<{ stdout: string; stderr: string }> {
+	options.env = { ...process.env, ...options.env }
```

Comment:
Neato!

Also isn't process just available (no import needed?): https://github.com/sonatype-nexus-community/vscode-iq-plugin/pull/187/files#diff-c95f6239f1374b3dc1be4cafeaf71efac187b103786c2466678b43b8c0e5d9bcR52-R54

Reply:
Good call - I've removed the import